instance_dedicated_osiris.tf

```diff
@@ -0,0 +1,13 @@
+resource "openstack_compute_instance_v2" "osiris-denbi" {
+  name            = "osiris-denbi.galaxyproject.eu"
+  image_name      = "Rocky 9"  # Assuming this matches de.NBI cloud image 361fa0da-86b0-48bf-840b-e79f7723b4a3
+  flavor_name     = "m1.large"
+  key_pair        = "cloud2"
+  security_groups = ["egress", "public-web2", "default", "public-ping"]
+
+  network {
+    name = "public"
```

Comment:
If we have this behind traefik, do you want this public for ssh?

Reply:
SSH access is somewhat hardened so it should be ok to have it open.

Is there any another alternative that still allows an occasional remote access (if needed) but doesn't expose ssh to the world?

Reply:
I can set up [Tailscale](https://tailscale.com/) and you can use that to ssh in the server :)

Reply:
>We don't need ssh access to run the platform and given the current ansible recipe I expect ssh access to be rare. Perhaps a little more frequent in the beginning as we polish any remaining hard edge.

We need it at least open to our internal subnets for ansible to work, but I would keep port 22 open with key authentication set up